tools/scripts/copy-rust-android.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
```

Comment:
Let's make the script fail upon any command exiting with a non-zero status code.

```suggestion
#! /bin/bash
set -e
```